.gitignore

```diff
@@ -32,3 +32,4 @@ build/
 ### VS Code ###
 .vscode/
 /src/main/resources/application.properties
+/src/main/resources/liquibase.properties
```

Comment:
Why do you exclude these properties from git?

Reply:
I exclude these properties from git because I keep original credentials for entering the database in it. But I provide examples of these properties with mock data for username and password.

Reply:
remove these files from gitignore, there is another way to hide credentials. You can use environment variables for it. 
I suggest you implement this logic at the end of project.